certbot/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM python:2-alpine
+FROM python:3.7-alpine
```

Just to keep the needle threaded, this is 3.7 because https://github.com/certbot/certbot/blob/v0.36.0/setup.py#L126-L133 only claims support for 3.7 (and we're still on 0.36.0, which might be more work to actually bump); I'm +1 on this update, but as elsewhere think we should take the opportunity to pin Alpine version:

```suggestion
FROM python:3.7-alpine3.10
```